2020-05-18 00:10:00.131  INFO 30477 --- [XNIO-1 I/O-1] lavalink.server.io.SocketServer          : {"op": "voiceUpdate", "guildId": "657066018337718333", "sessionId": "a67057f178f56674cdf90461f10dea01", "event": {"token": "2e81ea30f8883d9d", "guild_id": "657066018337718333", "endpoint": "brazil518.discord.media:80"}}
2020-05-18 00:10:01.208  INFO 30477 --- [XNIO-1 task-10] l.server.player.AudioLoaderRestHandler   : GET /loadtracks
2020-05-18 00:10:01.742  INFO 30477 --- [lava-daemon-pool-info-loader-2-thread-1] lavalink.server.player.AudioLoader       : Loaded playlist Search results for: hombres g chica cocodrilo
2020-05-18 00:10:01.880  INFO 30477 --- [XNIO-1 I/O-1] lavalink.server.io.SocketServer          : {"op": "play", "guildId": "657066018337718333", "track": "QAAArgIASUhvbWJyZXMgRyAtIEVsIEF0YXF1ZSBEZSBMYXMgQ2hpY2FzIENvY29kcmlsbyAoVmlkZW8gT2ZpY2lhbCkgVmlkZW9NdXNpYy4AC1ZpZGVvTXVzaWMuAAAAAAACt1AAC2Fzc0FVcEdXS2RFAAEAK2h0dHBzOi8vd3d3LnlvdXR1YmUuY29tL3dhdGNoP3Y9YXNzQVVwR1dLZEUAB3lvdXR1YmUAAAAAAAAAAA==", "noReplace": false, "startTime": "0"}
2020-05-18 00:12:21.361  INFO 30477 --- [XNIO-1 I/O-1] lavalink.server.io.SocketServer          : {"op": "stop", "guildId": "657066018337718333"}
2020-05-18 00:12:21.362  INFO 30477 --- [XNIO-1 I/O-1] lavalink.server.io.SocketServer          : {"op": "destroy", "guildId": "657066018337718333"}
2020-05-18 00:14:20.160 ERROR 30477 --- [XNIO-1 task-12] l.server.io.HandshakeInterceptorImpl     : Authentication failed from /172.105.3.178:33080
2020-05-18 00:38:14.039 ERROR 30477 --- [XNIO-1 task-14] l.server.io.HandshakeInterceptorImpl     : Authentication failed from /190.122.147.242:45274
2020-05-18 00:53:23.381 ERROR 30477 --- [XNIO-1 task-15] l.server.io.HandshakeInterceptorImpl     : Authentication failed from /143.255.242.185:19786
2020-05-18 02:49:05.877 ERROR 30477 --- [XNIO-1 task-16] l.server.io.HandshakeInterceptorImpl     : Authentication failed from /188.166.59.16:46184
2020-05-18 03:23:14.268  WARN 30477 --- [XNIO-1 task-1] l.s.config.RequestAuthorizationFilter    : Authorization missing for 109.234.153.132 on POST /azenv.php
2020-05-18 03:23:19.324 ERROR 30477 --- [XNIO-1 I/O-1] io.undertow.request                      : UT005071: Undertow request failed HttpServerExchange{ CONNECT check.best-proxies.ru:80}

java.lang.IllegalArgumentException: UT000068: Servlet path match failed
	at io.undertow.servlet.handlers.ServletPathMatchesData.getServletHandlerByPath(ServletPathMatchesData.java:83) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
	at io.undertow.servlet.handlers.ServletPathMatches.getServletHandlerByPath(ServletPathMatches.java:88) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
	at io.undertow.servlet.handlers.ServletInitialHandler.handleRequest(ServletInitialHandler.java:146) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
	at io.undertow.server.handlers.HttpContinueReadHandler.handleRequest(HttpContinueReadHandler.java:65) ~[undertow-core-2.0.26.Final.jar!/:2.0.26.Final]
	at io.undertow.server.Connectors.executeRootHandler(Connectors.java:376) ~[undertow-core-2.0.26.Final.jar!/:2.0.26.Final]
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:255) ~[undertow-core-2.0.26.Final.jar!/:2.0.26.Final]
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:136) ~[undertow-core-2.0.26.Final.jar!/:2.0.26.Final]
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:59) ~[undertow-core-2.0.26.Final.jar!/:2.0.26.Final]
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92) ~[xnio-api-3.3.8.Final.jar!/:3.3.8.Final]
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66) ~[xnio-api-3.3.8.Final.jar!/:3.3.8.Final]
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:88) ~[xnio-nio-3.3.8.Final.jar!/:3.3.8.Final]
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:561) ~[xnio-nio-3.3.8.Final.jar!/:3.3.8.Final]

2020-05-18 03:32:51.689 ERROR 30477 --- [XNIO-1 task-2] l.server.io.HandshakeInterceptorImpl     : Authentication failed from /180.244.235.13:30885
